### PR TITLE
[#5979]: Add ENS names to profile screen

### LINKED ISSUE #5979: Add ENS names to profile screen
### User Story

As a user, I want to discover ENS names from my profile screen so I can register an ENS address.

### Description

*Type*: Feature

*Summary*: Add navigation to the ENS DApp into the Status profile screen. 

#### Expected behavior
image

Clicking on `ENS names` should open `statusnames.eth`.

#### Actual behavior
No navigation yet.

### PR DESCRIPTION
**Pull request by alwx**
Fixes #5979

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

**Review by jeluard — APPROVED** [review]

**Review by janherich — APPROVED** [review]

File: src/status_im/ui/screens/profile/events.cljs [hunk 1]
```diff
@@ -87,3 +88,8 @@
  :show-tooltip
  (fn [_ [_ tooltip-id]]
    {:show-tooltip tooltip-id}))
+
+(handlers/register-handler-fx
+ :profile/open-ens-names-in-browser
```
Comment (yenda) [inline src/status_im/ui/screens/profile/events.cljs]: please use a name that reflects what happened in the UI not what is going to happen

File: src/status_im/ui/screens/profile/events.cljs [hunk 2]
```diff
@@ -87,3 +88,8 @@
  :show-tooltip
  (fn [_ [_ tooltip-id]]
    {:show-tooltip tooltip-id}))
+
+(handlers/register-handler-fx
```
Comment (yenda) [inline src/status_im/ui/screens/profile/events.cljs]: please move this to status-im.events

**Review by yenda — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/profile/events.cljs (hunk 2 above)
Comment (alwx) [reply src/status_im/ui/screens/profile/events.cljs]: why? what's the point?

File: src/status_im/events.cljs [hunk 3]
```diff
@@ -839,3 +839,8 @@
  :browser.ui/open-modal-chat-button-pressed
  (fn [cofx [_ host]]
    (browser/open-chat-from-browser cofx host)))
+
+(handlers/register-handler-fx
+ :profile/open-ens-names-in-browser
```
Comment (yenda) [inline src/status_im/events.cljs]: :profile.ui/ens-names-button-pressed

**Review by yenda — APPROVED** [review]

**Comment by asemiankevich** [comment]
I am curious if we want to introduce new screen leading to dead url (statusnames.eth seems to be down) . So question is: do we want to merge it as is or wait for the url up and runnung? 

cc @rachelhamlin @bgits

**Comment by asemiankevich** [comment]
i talked to @jeluard and the decision was made to wait for merge until the URL is up and running

**Comment by rachelhamlin** [comment]
I like that decision!

**Comment by alwx** [comment]
Ok. Yes, let's just wait.

**Review by vkjr — APPROVED** [review]

**Comment by jeluard** [comment]
@alwx Please update the ENS name to `names.statusnet.eth`  (and rebase :))

**Pull request merged**